File: src/Etapa1/C06EX07.java
package Etapa1;

import java.util.Scanner;

//FUNCAO: Calcula o valor da Participacao de Lucro de um funcionario de acordo com o seu salario
//AUTOR: ANGELO CARLOS MACHADO SANTOS

public class C06EX07 {
    public static void main(String[] args) {
        double salario, PLbruto, PLliquido, impRenda;
        Scanner teclado = new Scanner(System.in);
        System.out.println("Informe o valor do Sálario: ");
        salario = teclado.nextDouble();
        teclado.close();

        if (salario <= 300) {
            PLbruto = 500 + (salario * 0.7);
            impRenda = PLbruto * 0.25;
            PLliquido = PLbruto - impRenda;
            System.out.printf("A Participacao de Lucro liquida é de: R$%1.2f ", PLliquido);
        } else if (salario <= 1000) {
            PLbruto = 200 + (salario * 0.5);
            impRenda = PLbruto * 0.25;
            PLliquido = PLbruto - impRenda;
            System.out.printf("A Participacao de Lucro liquida é de: R$%1.2f ", PLliquido);
        } else {
            PLbruto = salario * 0.3;
            impRenda = PLbruto * 0.25;
            PLliquido = PLbruto - impRenda;
            System.out.printf("A Participacao de Lucro liquida é de: R$%1.2f ", PLliquido);
        }
    }
}

File: src/C06EX07.kt
import java.util.Scanner

fun main() {
    val teclado = Scanner(System.`in`)

    print("Informe o valor do Sálario: ")
    val salario: Double = teclado.nextDouble()
    teclado.close()

    if (salario <= 300) {
        val plBruto: Double = 500 + (salario * 0.7)
        val impRenda: Double = plBruto * 0.25
        val plLiquido: Double = plBruto - impRenda
        println("A Participacao de Lucro liquida é de: R$${"%.2f".format(plLiquido)}")
    } else if (salario <= 1000) {
        val plBruto = 200 + (salario * 0.5)
        val impRenda = plBruto * 0.25
        val plLiquido = plBruto - impRenda
        println("A Participacao de Lucro liquida é de: R$${"%.2f".format(plLiquido)}")
    } else {
        val plBruto = salario * 0.3
        val impRenda = plBruto * 0.25
        val plLiquido = plBruto - impRenda
        println("A Participacao de Lucro liquida é de: R$${"%.2f".format(plLiquido)}")
    }
}